CHANGELOG.md

```diff
@@ -13,6 +14,11 @@
 - The class parameter corosync::packages has been removed (was deprecated in
   0.8.0) (#282)
 
+### Deprecation notes
+
+We will remove support for Puppet <= 3.6.0 in two major releases of this module
```

Comment:
That is really a big change and we have no hurry to do so… I'd let people 2 major releases to adapt